packages/ui-components/src/components/QrCode/QrCodeContainer.tsx

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
Is this a copy of the one in `admin-panel`? If so, can we remove one of them? If this is only used in `admin-panel` I'd be in favour of removing this one.